shell/src/test/java/alluxio/cli/fsadmin/report/CapacityCommandTest.java

```diff
@@ -59,6 +59,7 @@ public void longCapacity() throws IOException {
       String output = new String(outputStream.toByteArray(), StandardCharsets.UTF_8);
       // CHECKSTYLE.OFF: LineLengthExceed - Much more readable
       List<String> expectedOutput = Arrays.asList("Capacity information for all workers: ",
+          "    Total vCPUs: 0",
```

Comment:
Update the cpu count to a non-zero number in `prepareLongInfoList()` and assert here?

Reply:
there might be other UTs you can add, like in `BlockMasterTest` check a registered worker will have CPU recorded and summed at the BlockMaster. I'll leave this up to your judgement, depending on the importance of this feature and how much we rely on real system tests instead.